examples/lighting-app/nrfconnect/prj_nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
Judging by nrf/samples/spm it seems it should be possible to define boards/board_name.conf file which would only contain values not contained in the main `prj.conf`. It would reduce some duplication, but I haven't tested that though.

Reply:
Found out the custom config won't be needed.